sdk/search/search-documents/test/public/utils/recordedClient.ts

```diff
@@ -77,11 +93,17 @@ export async function createClients<IndexModel extends object>(
       serviceVersion,
     })
   );
+  const openAIClient = new OpenAIClient(
```

Comment:
Good catch, I need to check if I can create an Azure OpenAI deployment through ARM. If not, this might need to be a recorded-only test? Or we can store the embeddings in another file.

Reply:
Looks like Jeremy and I had the same [comment ](https://github.com/Azure/azure-sdk-for-js/pull/26069#discussion_r1213767067)😊 so let's follow up here. I think provisioning an OpenAI resource could be overkill especially that in my experience it took hours! and it comes with a lot of limitations, I heard they allow literally only 1rpm at the moment if you are not on some special list. So for simplicity, I endorse skipping the test in live mode.

Reply:
Sounds good! I've gone ahead and disabled that test in live mode.

Reply:
What's the scenario we test here?  maybe this could be a sample showing the integration between search and openai? (not blocking PR just some thought)